src/test/java/org/broadinstitute/hellbender/tools/exome/segmentation/CopyRatioSegmenterUnitTest.java

```diff
@@ -9,50 +12,55 @@
 import java.util.Collections;
 import java.util.List;
 import java.util.Random;
+import java.util.stream.Collectors;
 import java.util.stream.IntStream;
 
 /**
  * Created by davidben on 6/6/16.
  */
 public final class CopyRatioSegmenterUnitTest {
+    private static final Random RANDOM = new Random(271);
```

Comment:
Same caution here.  After making the changes, please ensure that tests give back exactly identical results even if run in a different order.